### DISCUSSION TITLE
Osu MacOs doesn't start anymore

### DISCUSSION THREAD

**Discussion by Reckoner2** [ppy/osu#23065] [Q&A] [answered]
I updated the game as usual to the most recent version of Lazer and it was laggy, like REALLY laggy, I saw a video on YouTube about the new release and on the last point I saw ''Render'' it was automatic so I changed it like in the video to OpenGL and it just doesn't start the game anymore.

HELP PLEASE.

**Comment by bdach** [comment]
Please provide the logs as described in https://github.com/ppy/osu/discussions/15188.

To make the game work again you can either delete `framework.ini` from the game files, or if feeling particularly attached to your settings, remove the `Renderer = Vulkan` line from that file.

**Reply by Schweeeeeeeeeeeeeeee** [reply]
Should I provide logs to improve Vulkan renderer, or should I only send them to troubleshoot?

**Reply by bdach** [reply]
@Schweeeeeeeeeeeeeeee please open a separate discussion for whatever you want to do because this thread isn't about vulkan

**Reply by Reckoner2** [reply]
@bdach 
I really have no idea how to do that, I used the ~/Library/Application Support/osu/logs but it doesn't go anywhere and also, didn't find any framework.ini, I'm totally bad with computers.

**Comment by peppy** [comment, marked as answer]
The latest release should fix this one.

**Reply by Reckoner2** [reply]
@peppy I don't know if I had to do what @bdach said first (cause I couldn't do it, I'm stupid) but lazer still doesn't work.
